corelib/src/test/array_test.cairo

```diff
@@ -1,5 +1,6 @@
 use crate::iter::{IntoIterator, Iterator};
 use crate::test::test_utils::assert_eq;
+use crate::array::{EmptyFixedSizeArrayIntoIterator, FixedSizeArrayIntoIterator};
```

Comment:
demonstrates the requirement for an explicit import

Reply:
it isn't required - or at least it really shouldn't.
<!-- Reviewable comment -OFCVau78_shBZflm6bV:b-ffxvjy -->
<!-- Sent from Reviewable.io -->